tests/Microsoft.Identity.Test.Unit/PublicApiTests/ClientCredentialWithCertTest.cs

```diff
@@ -165,8 +168,7 @@ public async Task JsonWebTokenWithX509PublicCertSendCertificateTestAsync()
                     .WithAuthority(new System.Uri(ClientApplicationBase.DefaultAuthority), true)
                     .WithRedirectUri(TestConstants.RedirectUri)
                     .WithHttpManager(harness.HttpManager)
-                    .WithCertificate(certificate)
-                    .WithSendX5C(true)
+                    .WithCertificate(certificate, true)
```

Comment:
Probably easiest is to create a second app object + WithClientClaims to cover this.

Reply:
Added new unit test